### PR TITLE
Add "Helping out" document

### PR DESCRIPTION
**Pull request by bhrutledge**
Adding the "Jobs people can do" doc and relevant items from "BOSPY Organizing" Trello board to the site. We've already got page with "jobs" in the title, hence the new name.

### CODE REVIEW

File: helping.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+Search job postings for Python jobs in the Boston-area. Collect names and email addresses of people we can contact about sponsoring.
+
+Time: 1 hour/week; On-going; Can be shared; Casual; At-home.
+
+### Hosting wrangler - Mike
+
+Find companies to host our events. Reserve Microsoft NERD when it's available. Communicate with potential hosts about requirements. Evaluate potential spaces in person. Confirm dates for events, add to meetup.com.
+
+### Meadhall reserver - Ned
```
Comment (nedbat) [inline helping.md]: We should generalize from Meadhall.   Also, throughout, I'm not sure how to notate that some jobs simply aren't being done a lot, or at all at the moment.  Maybe we don't need to?

File: helping.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+Email with the Meadhall events person to reserve the private space for us after presentation nights. Check in with them the day before to be sure the reservation is in place.
+
+### Event list gatekeeper
+
+Keep track of other events people want us to announce to the group. Decide which events are appropriate for the list. Send the monthly email.
```
Comment (nedbat) [inline helping.md]: For example: "the monthly email."  What monthly email?

File: assets/css/main.scss [hunk 3]
```diff
@@ -26,6 +26,10 @@ h1, h2, h3, h4, h5, h6 {
     font-family: 'Roboto Slab', serif;
 }
 
+h2 {
+    margin: 2rem 0 1rem;
+}
+
```
Comment (nedbat) [inline assets/css/main.scss]: I applied this change already.

**Comment by bhrutledge** [comment]
@nedbat I did some reorganization, added some more jobs, pruned some descriptions, and added a table of contents. I'm happy to merge this if you are.

**Pull request merged**